### ISSUE TITLE
Topics page still has "Explore" in the title...

### ISSUE THREAD

**Issue by radinamatic** [learningequality/kolibri#1081]
## Summary

...did it get hardcoded somewhere?

This also has accessibility repercussions, so if the `H1` is `Topics`, it should be reflected in the `<title>`.

![explore - instant schools math - kolibri - google chrome_110](https://cloud.githubusercontent.com/assets/1457929/24023024/e411630c-0a66-11e7-8166-b73bd9da0771.png)

**Comment by jonboiser** [comment]
The relevant code is here:

https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/learn/assets/src/state/actions.js#L182

If you change 'Explore' to something like 'Foo', then the title will reflect that change.

The quick change would be to change that 'Explore' -> 'Topics', but it would not be localized, which is the direction things are moving at the Vue-component level.

@indirectlylit  Is there a way to get access to the intl stuff at the vuex level?

**Comment by indirectlylit** [comment]
Unfortunately at the moment we can only internationalize strings in .vue files. This is a known limitation that we've been putting off fixing, but we'll need to do it eventually.

Let's do the quick-fix for now. Created a new issue for the bigger task: https://github.com/learningequality/kolibri/issues/1144
